plugins/lookup/vault_list.py

```diff
@@ -64,22 +64,58 @@
   ansible.builtin.debug:
     msg: '{{ item }}'
   with_community.hashi_vault.vault_list:
-    - secret
+    - secret/metadata/path
```

Comment:
Needs to be 'secret/metadata' - will update.

Reply:
Done in https://github.com/ansible-collections/community.hashi_vault/pull/343/commits/857568ab7e72c9826cf56b5bb3691a4d950433d3

Reply:
it looks like https://github.com/ansible-collections/community.hashi_vault/commit/857568ab7e72c9826cf56b5bb3691a4d950433d3 only updated the module and not the lookup